LICENSE

```diff
@@ -45,7 +45,7 @@
       separable from, or merely link (or bind by name) to the interfaces of,
       the Work and Derivative Works thereof.
 
-      "Contribution" shall mean any work of authorship, including
+      "Contribution" shall mean any work of authorship, includin prog
```

Comment:
This line shouldn't be changed to be in sync with https://github.com/kubernetes-sigs/.github/blob/master/LICENSE#L48

Reply:
Done.